### PR TITLE
Random startup list

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #157

### CODE REVIEW

File: src/actions/show_random_startups.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const { mockCollection } = require('../../database/startup_schema.js');
+const random = require('mongoose-simple-random');
```
Comment (Karyum) [inline src/actions/show_random_startups.js]: is this being used somewhere in this file?

File: tests/server/index.js [hunk 2]
```diff
@@ -96,6 +96,16 @@ tape('Test the submit message page', (t) => {
   })
 })
 
+tape('Test for random startups page', (t) => {
+  supertest(server).get('/randomStartups').end((err,res) => {
+    t.error(err, 'No Error');
+    var testedClass = '<div class="dib mh1 tc">';
+    var count = (res.text.match(/<div class="dib mh1 tc">/g) || []).length;
```
Comment (Karyum) [inline tests/server/index.js]: you could change to those to const to keep it consistant

**Review by Karyum — CHANGES_REQUESTED** [review]
:+1: :maple_leaf:

File: tests/server/index.js (hunk 2 above)
Comment (Suuuuha) [reply tests/server/index.js]: you're right

**Comment by Suuuuha** [comment]
@Karyum changes made

**Review by Karyum — APPROVED** [review]
:+1:

**Pull request merged**